cypress/obj/timeLogging/timeLoggingPage.js

```diff
@@ -0,0 +1,35 @@
+class TimeLoggingPage {
+    checkUrl() {
+        cy.url().should('include', '/time-logging');
+    }
+
+    getPageTitle() {
+        return cy.get('.page__title');
+    }
+
+    getCalendar() {
+        return cy.get('.calendar');
+    }
+
+    getFormTile() {
+        return cy.get('.tile.form');
+    }
+
+    getUserNameTitle() {
```

Comment:
Methods like these could be put in` common.js` or` pageComponents.js` files, because this is shared between pages and if you would want to check username it would look weird if you have to use `timeLoggingPage` to do so.